package.json

```diff
@@ -14,14 +14,14 @@
     "storybook": "./web/scripts/run-storybook.sh",
     "storybook-smoke-test": "storybook dev -p 9002 -c web/.storybook --ci --smoke-test",
     "test-storybook": "NODE_TLS_REJECT_UNAUTHORIZED=0 test-storybook -c web/.storybook --url https://localhost:9002 --skipTags=skip-test --browsers=chromium",
-    "test": "jest",
-    "test-coverage": "jest --coverage && web/scripts/print-coverage-link.sh",
+    "test": "make dump-preset-roles && jest",
```

Comment:
What do you think about running this `make dump-preset-roles` from within `beforeAll` in the particular test file that needs it?

FWIW, it could even print the roles to stdout without having to interact with the filesystem at all. But that's more of a nit, I think just changing this to not be executed on every invocation of `test` or `tdd` is more important.

Reply:
Wow, I haven't thought about it. Although in this case, we'll probably run into test timeouts, as until Go adds caching linker artifacts, this takes a couple of seconds.